### DISCUSSION TITLE
Change request: update clearance heights of most stalls

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#18831] [Feature requests]
Related to #18827 

The cash machine is the most egregious example of a stall with the clearance set too low ... but while we're at it, why not fix a lot of other stalls?

The vast majority of stalls have a clearance height of 4 units, when they really should have three. (Those ones have no scenery placed in front of them in the screenshot below.)

As for the rest, from lowest to highest:

The Restrooms are all correctly set at 2 units, and don't need any change. (heart shrubs in front of them)

The sunglass hut, hat stall, T-shirt stall, and hot chocolate stall are all correctly set at 3 units, and don't need any change. (heart shrubs in front of them)

The coffee shop and the beef noodles stall are set at 3 units, but should probably be higher. The coffee shop's front sign looks taller than a 3-unit path, and the beef noodles stall actually clips through the path, which you can see in this screenshot. (red base blocks in front of them)

The first aid stall is set at 3 units and should probably stay that way, but the boundbox should be increased to 3 units as well so path supports no longer overlap its sign. (broken wheel in front)

Several more stalls are set at 4 units and should probably *stay* there. But all of them should have their boundboxes increased by one unit so that their top portions are no longer overlapped by path supports. (All of row 4 in the screenshot)

The two stalls in row 5 are set at 4 units, but should probably be even higher -- 5 units -- because they're just so tall. The support boundboxes should be increased as well. (red base blocks in front of them)

Screenshot:
![Forest Frontiers 2022-12-12 15-29-32](https://user-images.githubusercontent.com/26573303/207183309-65b69ff5-0ca7-4b25-8890-a07de74e888f.png)
(This screenshot was made using disabled clearance checks, to show how the stalls look when underneath their new minimum path height.)

If you want to play around with this yourself, here's the test park I was using:
[Stall Height Test.zip](https://github.com/OpenRCT2/OpenRCT2/files/10212905/Stall.Height.Test.zip)
(Rename to .park)
Using that could be more convenient for you, because it's already flattened and cleared, and already has absolutely every stall from the base game already present in it.

------

Probably too much for just one PR all at once, but it seems like it should be relatively easy to do with a series of smaller PRs.

If nothing else, at least the stalls with red base blocks in front of them should have their clearance heights increased. That part might be considered an actual bugfix, rather than a change.

**Comment by spacek531** [comment]
Paint code would have to change to change the support blocking height.
